scripts/aggregate_cells.py

```diff
@@ -65,29 +69,42 @@ def aggregate(sql_file, metadata_dir, barcode_platemap):
 
     # counting cells in each well and saving it as csv file
     print("Counting cells within each well")
-    cell_count_outfile = str(snakemake.output["cell_counts"])
     cell_count_df = single_cell_profile.count_cells()
 
-    print("Saving cell counts in: {}".format(cell_count_outfile))
+    print("Saving cell counts in: {}".format(cell_count_out))
```

Comment:
We'll likely eventually drop these print statements in favor of a logger. Just FYI!

Reply:
Correct! I'll right an issue in regards to this!

Reply:
Issue written! #9